domain-models-runtime-it/ramls/wrappedboolean.raml

```diff
@@ -0,0 +1,25 @@
+#%RAML 1.0
```

Comment:
Can you add a short boxing documentation to README.md with links to library/ramltojaxrs.raml and ramls/wrappedboolean.raml?

Reply:
Do you have a suggestion where at in the README would be appropriate?